application/src/main/java/uk/gov/hmcts/dm/controller/DocumentContentVersionController.java

```diff
@@ -27,10 +28,14 @@
 import uk.gov.hmcts.dm.service.DocumentContentVersionService;
 import uk.gov.hmcts.dm.service.StoredDocumentService;
 
+import javax.servlet.http.HttpServletResponse;
 import javax.validation.Valid;
+import java.io.IOException;
 import java.util.UUID;
 import java.util.stream.Collectors;
 
+import static org.apache.commons.lang.StringUtils.isBlank;
+
```

Comment:
Shall we use the later `org.apache.commons.lang3.StringUtils`

Reply:
Will change